Documentation/tls/tls-ingress.md

```diff
@@ -0,0 +1,58 @@
+# Enabling custom Tectonic Console TLS certificates
+
+Use Tectonic's [Ingress TLS module][ingress-module] to enable user provided Ingress certificates.
+
+The module does not contain any logic, but just passes user provided certificates from its input directly to its output. This is to prevent changing existing references to the `tls/ingress/self-signed` module, hence all `tls/ingress/*` modules share the same outputs.
```

Comment:
The certificates provided here are used only to secure the tls endpoint in front of the tectonic console and tectonic identity services. They are not used to secure your applications when you use the tectonic ingress controller. 

We need to clearly distinguish that we are securing an endpoint here. The tectonic console url. Not providing certs that are going to be used by folks that leverage the tectonic ingress controller for their own applications.

Reply:
We should also highlight that it is frequently necessary to make this change as folks that use a web browser will get a cert error if they don't change this.